husky_extensions/ckeditor-extension.js

```diff
@@ -140,20 +172,36 @@
                         CKEDITOR.plugins.add(name, plugin);
                     },
 
-                    addToolbarButton: function(toolbarName, button, icon) {
-                        if (!toolbar[toolbarName]) {
-                            toolbar[toolbarName] = [];
+                    addToolbarButton: function(toolbarName, button, icon, buttonRelations) {
+                        if (!availableToolbarSections[toolbarName]) {
+                            availableToolbarSections[toolbarName] = [];
                         }
 
-                        toolbar[toolbarName].push(button);
+                        availableToolbarSections[toolbarName].push(button);
```

Comment:
`availableToolbarSections[toolbarName]` is a bit confusing... does this object now contain toolbars or toolbar sections?